js/common/lib/env.ts

```diff
@@ -52,6 +53,13 @@ export declare namespace Env {
      * @defaultValue `false`
      */
     proxy?: boolean;
+
+    /**
+     * Set or get a boolean value indicating whether to enable training. If set to true, training will be enabled.
+     *
+     * @defaultValue `false`
+     */
+    enableTraining?: boolean;
```

Comment:
isn't this actually a const value that determined at compile time?

Reply:
what is the expected way for users to use this flag? could you share a snippet of example code to demonstrate that

Reply:
Oops, yes, you're right, the env.ts shouldn't contain the enableTraining flag